securitas/tests/unit/controller/test_password.py

```diff
@@ -16,6 +18,81 @@ def test_password_reset(client):
     assert page.title.string == 'Password Reset - The Fedora Project'
 
 
+@pytest.mark.vcr()
+def test_password_reset_user(client, logged_in_dummy_user):
+    """Test the redirect to the authed password reset"""
+    result = client.get('/password-reset')
+    assert result.location == f"http://localhost/user/dummy/password-reset"
```

Comment:
might be a silly question but will this always be the case? wondering will localhost be true across environments.

Reply:
As long as it's run from the Flask test runner (the `client` instance here) it should always be localhost.